src/serverMain/kotlin/com/studo/campusqr/endpoints/UserFrontend.kt

```diff
@@ -286,6 +343,16 @@ fun FlowContent.campusFooter(language: String, configs: Map<String, String>) {
       href = configs.getValue("imprintUrl")
       +configs.getValue("imprintText")
     }
+    liveCheckInLocationId?.let { locationId ->
+      a {
+        href = "/campus-qr/liveCheckIns?l=${locationId}"
+        target = "_BLANK"
```

Comment:
```suggestion
```

Reply:
Not sure about that. I think it's better to open it in a new tab since we have no back navigation in the UI